hack/check-skaffold-deps.sh

```diff
@@ -47,8 +47,9 @@ function check() {
     sort |\
     uniq > "$path_actual_dependencies"
 
-  # always add vendor directory
+  # always add vendor directory and VERSION file
   echo "vendor" >> "$path_actual_dependencies"
+  echo -e "VERSION\n$(cat $path_actual_dependencies)" > $path_actual_dependencies
```

Comment:
```suggestion
  echo "VERSION" >> "$path_actual_dependencies"
```

like for vendor?

Reply:
It needs to be added at the top of file, cause we are sorting these files [here](https://github.com/gardener/gardener/blob/7c3c3ef26103bc0143771276dfb907a857d8f252/hack/check-skaffold-deps.sh#L44-L48) and [here](https://github.com/gardener/gardener/blob/7c3c3ef26103bc0143771276dfb907a857d8f252/hack/check-skaffold-deps.sh#L44-L48). Sort keeps capital character string at top.

Reply:
What about sorting the whole file then once again after all contents were written? Then we don't need to care about such specialities.

Reply:
But isn't that an overhead?

Reply:
This is a "hack" script for local development productivity purposes. It does neither run in production, nor is the size of the file large enough to make us care about optimizing this script for efficiency.